### ISSUE TITLE
node_modules/mst-middlewares/dist/undo-manager.d.ts:17:16 - error TS2503: Cannot find namespace 'TypeFlags'.

### ISSUE THREAD

**Issue by xaviergonz** [mobxjs/mobx-state-tree#930]
I have:

* [x] I think something is not working as it should.
  * [x] I've checked documentation and searched for existing issues
  * [x] I've made sure your project is based on the latest MST version
  * [ ] Fork [this](https://codesandbox.io/s/jlnmjqr7xv) code sandbox or another minimal reproduction. 
  * [x] Describe expected behavior
  * [x] Describe observed behavior

```
node_modules/mst-middlewares/dist/undo-manager.d.ts:17:16 - error TS2503: Cannot find namespace 'TypeFlags'.
```

latest mobx, mobx-state-tree and mst-middlewares
ts 2.9.2
all strict options set to true

**Comment by fruitraccoon** [comment]
To work around this issue, I downgraded `mst-middlewares` from version `3.0.0` to `2.2.0`.

**Comment by mweststrate** [comment]
Released as 3.0.2
